src/components/cylc/Mutation.vue

```diff
@@ -18,55 +18,57 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <template>
     <v-card class="d-inline-block pa-4">
       <!-- the mutation title -->
-      <h3 :style="{ 'text-transform': 'capitalize' }">
-        {{ mutation._title }}
-      </h3>
-
-      <!-- the mutation description -->
-      <v-expansion-panels
-        variant="accordian"
-        v-bind="extendedDescription ? { hover: true } : { readonly: true }"
-      >
-        <v-expansion-panel
-          class="mutation-desc"
-          elevation="0"
+      <v-card-title>
+        <h3 :style="{ 'text-transform': 'capitalize' }">
+          {{ mutation._title }}
+        </h3>
```

Comment:
I've noticed the `h3` element and the text-transform are not really needed 

```suggestion
      <v-card-title class="py-3">
        {{ mutation._title }}
```